config/routes.rb

```diff
@@ -210,6 +210,13 @@
       end
     end
 
+    resource :student_accounts, only: [], controller: :student_accounts do
+      collection do
+        get :actions
```

Comment:
I would call this `reports` instead of `actions`. `action` in Rails has a specific/particular meaning, and it's not **really** an "action"—IMO more of a query or a report. I suppose unless you think of it as **executing** a query 🤷‍♂ .

Reply:
I tried this and it did not work. I reverted the code.

Reply:
For this to work, would have had to change this to `get :reports` and change the controller method from `def actions` to `def reports` and change `actions.html.erb` to `reports.html.erb`.